build.sbt

```diff
@@ -28,6 +28,9 @@ commands := commands.value.filterNot { command =>
   }
 }
 
+// pekkoInlineEnabled must be set to false when this is backported to 1.0.x branch
+ThisBuild / pekkoInlineEnabled := true
```

Comment:
Technically this part is not needed since its already enabled by default, I was planning to add

```scala
// pekkoInlineEnabled must be set to false when this is backported to 1.0.x branch
ThisBuild / pekkoInlineEnabled := false
```

When doing the cherry pick backport to `1.0.x`.

I can handle the backports if you want, this is not hard to do.

Reply:
I prefer to keep this explicit. There are a few contributors and they are not all up to date with the subtleties of this sbt plugin.

Reply:
Fair enough, I won't fight over this. We can always change it later once the storm of the 1.1.x releases are over